.github/ISSUE_TEMPLATE/05-performance.yml

```diff
@@ -121,48 +121,30 @@ body:
         - TVM
         - Vitis AI
         - Other / Unknown
+      multiple: yes
     validations:
       required: true
   - type: input
-    id: ha-version
+    id: ep-version
     attributes:
-      label: Hardware Acceleration Version
+      label: Execution Provider Library Version
       placeholder: ex. 2022.1
     validations:
-      required: true
-  - type: dropdown
+      required: false
+  - type: textarea
     id: model
     attributes:
       label: Model Type
-      options:
-        - BERT
-        - GPT2
-        - Hugging Face
-        - Longformer
-        - T5
-        - Other / Unknown
+      placeholder: Include your model type and attach the model file (if possible).

```

Comment:
Looks like there isn't a way to attach files when filing the issue, so suggest to reword this to "Add details about the model architecture and if possible, share the model file"

Reply:
There is a way to attach files when filing an issue! Each text box has a note that says "Attach files by dragging & dropping, selecting, or pasting them." Should I still reword?